docs/quick_start/data_sync/mysql_clickhouse_example.md

```diff
@@ -0,0 +1,142 @@
+---
+title: MySQL to ClickHouse Example
+sidebar_position: 1
+---
+
+Here we use an example to introduce how to use Apache InLong creating `MySQL -> ClickHouse` data synchronization.
+
+## Resource
+
+- Download the latest released [Apache InLong 1.9.0 installation package](https://downloads.apache.org/inlong/1.9.0/apache-inlong-1.9.0-bin.tar.gz)
+- Prepare Docker, Docker Compose environment
+
+## Deployment
+### Install Apache InLong
+
+Decompression apache-inlong-1.9.0-bin.tar.gz
+
+``` shell
+# Go to installation directory
+cd docker/docker-compose
+# Start
+docker-compose up -d
+```
+
+### Install ClickHouse
+```shell
+docker run -d --rm --net=host --name clickhouse -e CLICKHOUSE_USER=admin -e CLICKHOUSE_PASSWORD=inlong -e CLICKHOUSE_DEFAULT_ACCESS_MANAGEMENT=1 clickhouse/clickhouse-server:22.8
+```
+
+## Cluster Initialize
+When all containers are successfully started, you can access the InLong dashboard address http://localhost, and use the following default account to log in.
+```
+User: admin
+Password: inlong
+```
+
+### Create Cluster Tag
+Click [Clusters] -> [ClusterTags] -> [Create] on the page to specify the cluster label name and person in charge.
+
+![Create Cluster Tag](img/mysql_clickhouse/cluster_tag.png)
+
+:::caution
+Since each component reports the ClusterTags as `default_cluster` by default, do not use other names.
+:::
+
+### Register Pulsar Cluster
+Click [Clusters] -> [Cluster] -> [Create] on the page to register Pulsar Cluster.
+
+![Create Pulsar](img/mysql_clickhouse/pulsar.png)
+
+:::note
+The ClusterTags selects the newly created `default_cluster`, the Pulsar cluster deployed by docker:
+
+Service URL is `pulsar://pulsar:6650`, Admin URL is `http://pulsar:8080`.
+:::
+
+### Register ClickHouse DataNodes
+
+Click [DataNodes] -> [Create] on the page to register ClickHouse DataNodes.
+
+![Create Clickhouse Datanode](img/mysql_clickhouse/clickhouse_datanode.png)
```

Comment:
please provide a english version for this pic

Reply:
Done.